config/settings.yml

```diff
@@ -12,6 +12,9 @@ plugins:
     default: ''
   github_permalinks_enabled:
     default: false
+  github_permalinks_exclude:
+    default: "README.md|discourse/discourse/docs/*"
```

Comment:
It seems slightly unusual to me to allow this plugin to point to any repo, yet have a discourse specific default in there. I think it would make more sense for us to configure that specific exclusion on our install.